types/sanctuary/index.d.ts

```diff
@@ -166,6 +166,14 @@ declare namespace Sanctuary {
       (m: ReadonlyArray<A>): A[];
       (m: Foldable<A>): Foldable<A>;
     };
+    reject <A>(pred: Predicate<A>): {
+      (m: ReadonlyArray<A>): A[];
+      (m: Foldable<A>): Foldable<A>;
+    };
+    rejectM<A>(pred: Predicate<A>): {
+      (m: ReadonlyArray<A>): A[];
+      (m: Foldable<A>): Foldable<A>;
+    };
```

Comment:
Which Sanctuary version are you targeting, @lfarroco? I don't think Sanctuary has ever exported `rejectM`.

Reply:
My bad! I saw `filterM` in the list of definitions and I supposed that there would be a `rejectM` counterpart for it. Removing.

Reply:
Please update the first line of the file to reflect the Sanctuary version you are targeting.

Reply:
Cool! After checking the changelogs between 0.14 and the latest version, I found several new and removed methods that need be to be added (the ones in this PR are just some few that I stumbled upon).